docs/getting-started/install-endpoint.asciidoc

```diff
@@ -94,27 +94,35 @@ If the prompt does not appear because you're using a version before macOS Big Su
-3. By default, the **Malware Protections Enabled** toggle is on. To disable malware protection, switch the toggle off. Malware protection levels are as follows:
-* **Detect**: Detects malware on the host and generates an alert. The agent will **not** block malware. You must pay attention to and analyze any malware alerts that are generated.
-* **Prevent** (Default): Detects malware on the host, blocks it from executing, and generates an alert.
-4. Click **Save** to save changes to the Policy.
-5. On the dialog that appears, click **Save and Deploy changes**. If successful, a "Success" confirmation appears in the lower-right corner.
+3. By default, the **Malware Protections Enabled** toggle is on, with host notifications enabled or disabled based on the protection level. To disable malware protection, switch the toggle off. Malware protection levels are as follows:
+
+* **Detect**: Detects malware on the host and generates an alert. The agent will **not** block malware. You must pay attention to and analyze any malware alerts that are generated. Notifications do not appear by default. Select the **Notify User** option to enable them.
+* **Prevent** (Default): Detects malware on the host, blocks it from executing, and generates an alert. Notifications appear by default. Deselect the **Notify User** option to disable them.
++
+TIP: Platinum and enterprise users can customize these notifications using the `Elastic Security {action} {filename}`syntax. 
+
+4. In the **Settings** section, review the types of events you would like collected based on your host's needs. All event types are collected by default. To not include an event in your collection, deselect that event.
+5. (Windows Only) On Window's 10 or above, you can use Elastic Security as your antivirus software. For more information, see <<register-as-antivirus, Register as antivirus>>
+6. Click **Save** to save changes to the Policy.
+7. On the dialog that appears, click **Save and Deploy changes**. If successful, a "Success" confirmation appears in the lower-right corner.
 
 [role="screenshot"]
 image::images/install-endpoint/malware-protection.png[]
 
+[discrete]
+[[register-as-antivirus]]
 ## Register as antivirus (Windows Only)
 
 If you download the Elastic Agent on Windows 10 or above, you can configure Elastic Security as your antivirus software by doing the following:
 
 1. Go to **Fleet** > **Agents**. A list of hosts with the Elastic Agent appears. 
 2. Select your desired host from the list.
 3. From the host's page, find your named **Endpoint** integration and select it. 
-4. On the Integration Edit page, look for the **Protections** section and find **Type: Register as antivirus**. Toggle this option to enable. 
+4. On the Integration Edit page, look for the **Settings** section and find **Type: Register as antivirus**. Toggle this option to enable. 
 +
 [role="screenshot"]
 image::images/register-as-antivirus.png[]
```

Comment:
I'm a little lost here. Why do they need to do all this when they can select the option at the bottom? I can understand if they're in the Fleet app, but if they are following this page, they'll already be on the Policy config page. 

![image](https://user-images.githubusercontent.com/57149392/105898512-8944be80-5fe7-11eb-8f7c-17a3f38c83ff.png)

Reply:
These were written in a separate PR initially, but I was writing the content I discovered that this should be grouped together. Consolidating these in my next commit.